index.html

```diff
@@ -196,9 +286,30 @@ <h1>About
         <li class='menu'>Skills<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
       </ul>
 
+    </div>
+
+  </section>
+  <section class="contactform">
+    <div class="container"></div>
+    <div class="content-form">
+      <h3>Contact me</h3>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it </p>
+      <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/xgerlrpz"
+        method="POST">
+        <fieldset id="fs-frm-inputs">
+          <input type="text" name="name" id="full-name" placeholder="Full name" maxlength="30" required>
+          <input type="email" name="_replyto" id="email-address" placeholder="Email address" required>
+          <textarea rows="5" name="message" id="message" placeholder="Enter text here" maxlength="500"
+            required></textarea>
+          <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
+        </fieldset>
+        <input type="submit" value="Get in touch" class="submit-button">
```

Comment:
- [ ] Great job on your button, it looks just like the original, however, it is not yet well-positioned as the original one, see the difference below: 👇 
- Your button
![image](https://user-images.githubusercontent.com/55465133/129768456-78abebe6-d7fd-4210-a295-0c4ed1f1e1ae.png)

- The original button
![image](https://user-images.githubusercontent.com/55465133/129769031-3d8271ab-d7ee-4caa-917c-ed513c49d226.png)

Your button seems to be positioned to the center, but the original is to the left.
Please let us fix this button position to look like the original one 😉